File: solutions/python/line-up/1/line_up.py
def line_up(name, number):
    result = ""
    str_number = str(number)
    if str_number.endswith("1") and not str_number.endswith("11"):
        result = f"{name}, you are the {str_number}st customer we serve today. Thank you!"
    elif str_number.endswith("2") and not str_number.endswith("12"):
        result = f"{name}, you are the {str_number}nd customer we serve today. Thank you!"
    elif str_number.endswith("3") and not str_number.endswith("13"):
        result = f"{name}, you are the {str_number}rd customer we serve today. Thank you!"
    else:
        result = f"{name}, you are the {str_number}th customer we serve today. Thank you!"
    return result

File: solutions/javascript/line-up/1/line-up.js
//
// This is only a SKELETON file for the 'Line Up' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const format = (name, number) => {
  const lastTwo = number % 100;
  const lastOne = number % 10;

  let suffix = "th";
  if ( lastTwo < 11 || lastTwo > 13){
    if (lastOne === 1) suffix = 'st';
    else if (lastOne === 2) suffix = 'nd';
    else if (lastOne === 3) suffix = 'rd';
  }
  return `${name}, you are the ${number}${suffix} customer we serve today. Thank you!`;
};
